tests/from.bats

```diff
@@ -182,3 +182,30 @@ load helpers
   [[ "$output" =~ 41943040 ]]
   buildah rm $cid
 }
+
+@test "from volume test" {
+  cid=$(buildah from --volume=${TESTDIR}:/myvol --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  run buildah run $cid -- cat /proc/mounts
+  echo $output
+  [ "$status" -eq 0 ]
+  [[ "$output" =~ "/myvol ext4 rw," ]]
+  buildah rm $cid
+}
+
+@test "from volume ro test" {
+  cid=$(buildah from --volume=${TESTDIR}:/myvol:ro --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  run buildah run $cid -- cat /proc/mounts
+  echo $output
+  [ "$status" -eq 0 ]
+  [[ "$output" =~ "/myvol ext4 ro," ]]
+  buildah rm $cid
+}
+
+@test "from shm-size test" {
+  cid=$(buildah from --shm-size=80m --pull --signature-policy ${TESTSDIR}/policy.json alpine)
```

Comment:
This is fine, although I am not sure it is that precise.  You probably should just look at /dev/shm